cwms-data-api/src/main/java/cwms/cda/formatters/Formats.java

```diff
@@ -146,19 +147,10 @@ private String getFormatted(ContentType type, CwmsDTOBase toFormat) throws Forma
 
     }
 
-    private OutputFormatter getOutputFormatter(ContentType type, Class<? extends CwmsDTOBase> klass) {
-        OutputFormatter outputFormatter = null;
-        Map<Class<CwmsDTO>, OutputFormatter> contentFormatters = formatters.get(type);
-        if (contentFormatters != null) {
-            outputFormatter = contentFormatters.get(klass);
-        }
-        return outputFormatter;
-    }
-
     private String getFormatted(ContentType type, List<? extends CwmsDTOBase> dtos, Class<?
             extends CwmsDTOBase> rootType) throws FormattingException {
         for (ContentType key : formatters.keySet()) {
-            logger.finest(() -> key.toString());
+            logger.finest(key::toString);
```

Comment:
can we use the is level enabled trick with JUL to just shortcut this loop entirely?

Reply:
oh, I like the old way better.  I'm going to revert that line.

Reply:
Check if it still exists on master though, did remove a fair amount from this class.